lib/ui/chat/chat_info/group_chat_info.dart

```diff
@@ -149,61 +160,141 @@ class _GroupChatInfoState extends ConsumerState<GroupChatInfo> {
             ),
           ),
           Gap(16.h),
-          Text(
-            'Group Description:',
-            style: context.textTheme.bodyMedium?.copyWith(
-              color: context.colors.mutedForeground,
-              fontSize: 16.sp,
-              fontWeight: FontWeight.w600,
+          if (groupDescription.isNotEmpty) ...[
+            Text(
+              'Group Description:',
+              style: context.textTheme.bodyMedium?.copyWith(
+                color: context.colors.mutedForeground,
+                fontSize: 16.sp,
+                fontWeight: FontWeight.w600,
+              ),
             ),
-          ),
-          Text(
-            groupDetails?.description ?? '',
-            style: context.textTheme.bodyMedium?.copyWith(
-              color: context.colors.primary,
-              fontSize: 14.sp,
+            Gap(4.h),
+            Text(
+              groupDetails?.description ?? '',
+              style: context.textTheme.bodyMedium?.copyWith(
+                color: context.colors.primary,
+                fontSize: 14.sp,
+              ),
             ),
-          ),
-          Gap(32.h),
+          ] else ...[
+            GestureDetector(
+              onTap: isAdmin ? () => Routes.goToEditGroup(context, widget.groupId) : null,
+              child: Text(
+                isAdmin ? 'Add Group Description...' : 'No Group Description',
+                style: context.textTheme.bodyMedium?.copyWith(
+                  color: context.colors.mutedForeground,
+                  fontSize: 16.sp,
+                  fontWeight: FontWeight.w600,
+                ),
+              ),
+            ),
+          ],
+          if (isAdmin) ...[
+            Gap(24.h),
+            Padding(
+              padding: EdgeInsets.symmetric(horizontal: 24.w),
+              child: WnFilledButton(
+                size: WnButtonSize.small,
+                visualState: WnButtonVisualState.secondary,
+                label: 'Edit Group',
+                onPressed: () => Routes.goToEditGroup(context, widget.groupId),
+              ),
+            ),
+          ],
           // TODO: Reenable when we have a search and mute features
-          // Row(
-          //   spacing: 12.w,
-          //   children: [
-          //     Expanded(
-          //       child: WnFilledButton.icon(
-          //         visualState: WnButtonVisualState.secondary,
-          //         icon: SvgPicture.asset(
-          //           AssetsPaths.icSearch,
-          //           width: 14.w,
-          //           colorFilter: ColorFilter.mode(context.colors.primary, BlendMode.srcIn),
+          // Gap(36.h),
```

Comment:
I prefer to delete unused code rather than comment it. Commented-out code tends to go stale, and we can easily end up with comments that are  outdated, non-working code.